src/NzbDrone.Mono/Disk/DiskProvider.cs

```diff
@@ -89,11 +97,6 @@ public override void CopyPermissions(string sourcePath, string targetPath, bool
                 {
                     Syscall.chmod(targetPath, srcStat.st_mode);
                 }
-
-                if (includeOwner && (srcStat.st_uid != tgtStat.st_uid || srcStat.st_gid != tgtStat.st_gid))
```

Comment:
Keep this in. I think it was added to deal with a particular edgecase when copying folders.
It'll still not be effective in much scenarios, lacking appropriate permissions, but it's not user facing.

Reply:
Actually... this was the issue https://github.com/Sonarr/Sonarr/issues/3161
'Implemented' in v3 last year.

I'm torn a bit between tearing it out and leaving it in .... 😕 

... yup, tear it out.